tree-sitter-crabscript/tree-sitter.json

```diff
@@ -0,0 +1,31 @@
+{
+  "grammars": [
+    {
+      "name": "crabscript",
+      "camelcase": "Crabscript",
+      "scope": "source.crabscript",
+      "path": ".",
+      "file-types": [
+        "crab"
+      ],
+      "highlights": "queries/highlights.scm",
+      "injection-regex": "^(crab|crabscript)$"
+    }
+  ],
+  "metadata": {
+    "version": "0.0.1",
+    "license": "MIT",
+    "description": "Crabscript grammar for tree-sitter",
+    "links": {
+      "repository": "https://github.com/tree-sitter/tree-sitter-crabscript"
```

Comment:
💡 Fix this to this `https://github.com/IpFruion/crabscript/tree-sitter-crabscript`. Eventually it would be nice to be part of the `tree-sitter` project.